CHANGES.rst

```diff
@@ -47,6 +47,16 @@ ramp_fitting
   group, the timing for these ramps is not group time.  These adjusted times
   are now used. [#7612, spacetelescope/stcal#173]
 
+tweakreg
```

Comment:
These entries need to be moved to the top of the log, in the unreleased section.